.gitignore

```diff
@@ -14,7 +14,7 @@
 *.orig
 
 ## Directory-based project format:
-# .idea/
+.idea/
```

Comment:
although the folder was exluded, the files still appear in the pullrequest. These should be deleted if excluded